modules/visual_script/visual_script_flow_control.cpp

```diff
@@ -34,9 +34,7 @@
 #include "core/io/resource_loader.h"
 #include "core/os/keyboard.h"
 
-//////////////////////////////////////////
 ////////////////RETURN////////////////////
```

Comment:
```suggestion
// RETURN
```

And the same for the other places in this file.